miguel/block-one.js

```diff
@@ -0,0 +1,94 @@
+// In this file we are going to write the problems. Let's keep all in english to practice that as well.
+
+// 1. Create a function that returns me an array of 10 names with alphabetically order.
+
+// 2. Given an array of numbers, I want the sum of all of those numbers:
+    // Example: [10, 6, 8] be equal 24.
+
+// 3. You have an array of person objects, where each object contains a name and age.
```

Comment:
do this excercise please

Reply:
I'm missing this exercise at the momment, will finish it later today June 26th